apps/src/utils.js

```diff
@@ -110,7 +110,9 @@ exports.stripQuotes = function(inputString) {
  * Defines an inheritance relationship between parent class and this class.
  */
 Function.prototype.inherits = function (parent) {
-  this.prototype = _.create(parent.prototype, { constructor: parent });
+  this.prototype = Object.create(parent.prototype);
+  this.prototype.constructor = this;
+  this.superPrototype = parent.prototype;
```

Comment:
`super` is a reserved word in Javascript (for use in a future version,  I guess?) and superPrototype seemed both like a fairly accurate description of what you're getting and unlikely to cause collisions.  I would be fine with using something like `__base` instead, if you like that better - though the native idiom doesn't seem to use underscores much.